Scripts/script-Malwr.yml

```diff
@@ -0,0 +1,262 @@
+                  res = 'Error with the file.'
+
+          return res, soup
+
+      def get_status(self, analysis_id):
+          s = self.session
+          req = s.get(self.url + STATUS_URL % (analysis_id, ), headers=self.headers)
+          soup = BeautifulSoup(req.content, 'html.parser')
+          submission_links = MalwrAPI.find_submission_links(req)
+          if len(submission_links) > 0:
+              status = 'complete'
+          elif 'The analysis is still pending' in str(soup):
+              status = 'pending'
+          else:
+              status = 'error'
+
+          return status, submission_links[0], soup
+
+      def get_result(self, analysis_id):
+          analysis_status, _, _ = self.get_status(analysis_id)
+          if analysis_status != 'complete':
+              status = 'pending'
+              soup = None
+              is_malicious = None
+              md5 = None
+          else:
+              status = 'complete'
+              s = self.session
+              req = s.get(self.url + RESULT_URL % (analysis_id, ), headers=self.headers)
+              soup = BeautifulSoup(req.content, 'html.parser')
+              is_malicious = 'malicious' in str(soup)
+              soup_str = str(soup)
+              start_index = soup_str.find(MD5_PREFIX_STR)
+              if start_index == -1:
+                  md5 = None
+              else:
+                  start_index += len(MD5_PREFIX_STR)
+                  md5 = soup_str[start_index : start_index + 32]
+
+          return status, is_malicious, soup, md5
+
+
+  commandAction = demisto.args()['commandAction']
+
+  if commandAction not in SUPPORTED_COMMANDS:
+      demisto.results('Error: illegal command: %s. Supported commands are: %s' % (commandAction, SUPPORTED_COMMANDS))
+      sys.exit(0)
+
+  if 'username' in demisto.args() and 'password' in demisto.args():
+      username = demisto.args()['username']
+      password = demisto.args()['password']
+  else:
+      username = None
+      password = None
+
+  malwr = MalwrAPI(
+      url=MAIN_URL,
+      username=username,
+      password=password
+  )
+
+  entry = {
+      'Type': entryTypes['note'],
+      'ContentsFormat' : formats['text'],
+      'ReadableContentsFormat' : formats['text']
+  }
+
+  if commandAction == 'Submit':
+      fileId = demisto.args()['commandArgument']
+      filepathResult = demisto.executeCommand('getFilePath', {'id': fileId})
+      if 'path' not in filepathResult[0]['Contents']:
+          demisto.results('Error: entry %s is not a file.' % (fileId, ))
+          sys.exit(0)
+
+      res, soup = malwr.submit_sample(filepath=filepathResult[0]['Contents']['path'])
+      if 'analysis_link' in res:
+          analysisId = res['analysis_link'].split('/')[-2]
+
+          message = 'File submitted: %s%s\n' % (MAIN_URL, res['analysis_link'])
+          message += 'MD5: %s\n' % (res['md5'], )
+          message += 'Analysis ID: %s' % (analysisId, )
+
+          entry['Contents'] = str(soup)
```

Comment:
soup is the HTML page, e.g.:
![image](https://user-images.githubusercontent.com/26041559/28749335-43fc20b6-74cc-11e7-8332-e94f88feb866.png)

How can you convert this to JSON?